packages/ubuntu_provision/lib/src/locale/locale_page.dart

```diff
@@ -33,40 +39,62 @@ class LocalePage extends ConsumerWidget with ProvisioningPage {
       nextFocusNode: nextFocusNode,
       bottomBar: WizardBar(
         trailing: [
-          NextWizardButton(
-            focusNode: nextFocusNode,
-            onNext: () async {
-              final locale = model.locale(model.selectedIndex);
-              await model.applyLocale(locale);
-              await tryGetService<TelemetryService>()
-                  ?.addMetric('Language', locale.languageCode);
+          Focus(
+            onKeyEvent: (node, event) {
+              if (event is KeyDownEvent &&
+                  event.logicalKey == LogicalKeyboardKey.tab &&
+                  !HardwareKeyboard.instance.isShiftPressed) {
+                // Tab from Next button goes back to list
+                listFocusNode.requestFocus();
+                return KeyEventResult.handled;
+              }
+              return KeyEventResult.ignored;
             },
```

Comment:
This change seems to work identically if this is removed, any reason this was included?